src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -1131,6 +1138,20 @@ private bool TryProcessDiagnosticSourceStartEvents(TraceEvent data)
             return false;
         }
 
+        private void FixAndProcessAzureMonitorOpenTelemetryEvents(TraceEvent data)
```

Comment:
Consider removing "AzureMonitor" from the name. This EventSource comes from the "base" OpenTelemetry SDK -- not the Azure Monitor distro.

Reply:
Of course.

Reply:
Fixed.